rfcs/0004-pablo-distribution.adoc

```diff
@@ -0,0 +1,480 @@
+= Design Proposal: Pablo Fees & Staking Rewards Distribution
+:math:
+:stem:
+:toc:
+:toclevels: 4
+:sectnums:
```

Comment:
in MD there is pluging to insert toc, but yeah, not sure how automated it is. other, may be github md supports toc too

Reply:
I'm pretty sure I have tried most md plugins in the past, very hard to get a nice numbered/linked/multilevel toc with md.

Reply:
I understand your concern, but you may think of this document as source code of the actual document like latex source code of a document. This when converted, creates the actual document, that is a markdown file. Some others can decide to edit the markdown file generated later when the doc is more stable, which is totally fine. I've been following this workflow for some years now.